mcgill-augmented-reality-frontend/screens/SignupScreen.js

```diff
@@ -0,0 +1,116 @@
+import { React, useState } from 'react';
+import { StyleSheet, View, TouchableOpacity, TouchableWithoutFeedback, Text, TextInput, Keyboard } from 'react-native';
+import { registerUser } from '../api';
+
+function SignupScreen({ navigation }) {
+
+    const [name, onChangeName] = useState('');
+    const [email, onChangeEmail] = useState('');
+    const [password, onChangePassword] = useState('');
+
+    function handlePress() {
+        registerUser(email, password)
+        .then(res => {
+            if (res.status !== 200) {
+                console.log(res.error())
+            }
+        })
+        // navigation.navigate('nextScreenName');
+    }
+
+    return (
+        <TouchableWithoutFeedback onPress={() => Keyboard.dismiss()}>
+            <View style={styles.container}>
+                <Text style={styles.signupText}>Signup</Text>
+                <View style={styles.spaceSmall}/>
+                <Text style={styles.infoText}>
+                    Please fill out the following details to create an account
+                </Text>
+                <View style={styles.spaceLarge}/>
+                <TextInput 
+                    style={styles.input} 
+                    onChangeText={onChangeName}
+                    value={name}
+                    placeholder='Full Name' 
+                />
+                <View style={styles.spaceSmall}/>
+                <TextInput 
+                    style={styles.input} 
+                    onChangeText={onChangeEmail} 
```

Comment:
We should probs have some email and password validation so the user can't just put anything.